src/otx/v2/adapters/torch/lightning/dataset.py

```diff
@@ -64,8 +64,8 @@ def _build_dataset(
             if subset == "predict":
                 otx_dataset = self.dataset_entity
             else:
-                otx_dataset = self.dataset_entity.get_subset(str_to_subset_type(subset))
-            if len(otx_dataset) < 1:
+                otx_dataset = self.dataset_entity.get(str_to_subset_type(subset), None)
+            if not otx_dataset or len(otx_dataset) < 1:
```

Comment:
How about this?
```suggestion
            if otx_dataset is None or len(otx_dataset) == 0:
```

Reply:
Thank you. I applied your suggestion at 0e4de99acf5e9b1e54d3375cba358c4c02d964b2.